src/main/java/frc/team2412/robot/commands/autonomous/JackStealFourBallAutoCommand.java

```diff
@@ -84,6 +86,8 @@ public JackStealFourBallAutoCommand(DrivebaseSubsystem drivebaseSubsystem, Intak
                 new Follow2910TrajectoryCommand(drivebaseSubsystem, PATH_2),
                 new Follow2910TrajectoryCommand(drivebaseSubsystem, PATH_3),
                 new Follow2910TrajectoryCommand(drivebaseSubsystem, PATH_4),
-                new IndexSpitCommand(indexSubsystem));
+                new ParallelCommandGroup(
+                        new IndexSpitCommand(indexSubsystem),
+                        new IntakeSetOutCommand(intakeSubsystem)));
```

Comment:
I'm pretty sure this is equivalent (either way, the command doesn't finish because `IndexSpitCommand` doesn't finish) but clearer.
```suggestion
                new IntakeSetOutCommand(intakeSubsystem),
                new IndexSpitCommand(indexSubsystem));
```